nntrainer/layers/layer_devel.h

```diff
@@ -53,6 +53,16 @@ enum class InPlaceType {
                     layers ahead of it to be in-place */
 };
 
+/**
+ * @brief Emum class for the direction of inplace
+ *
+ */
+enum class InPlaceDirection {
+  NONE,  /**< default value */
```

Comment:
what does `NONE` indicate? does it mean it is out of place?

Reply:
@djeong20 The default value is set to `NONE` to use it as a default when the inplace option is turned off or when the inplace option is turned on but it's a unary operation, then there's no need to express direction using `LEFT` or `RIGHT`. However, if expressing it this way causes confusion, it would be fine to leave only `LEFT` and `RIGHT`.

Reply:
sorry, I'm a bit confused. when `is_inplace` is false, what should `getInPlaceDirection()` return?

Reply:
@djeong20 if it is called when `is_inplace` is false, then it will return `NONE`.

Reply:
> @djeong20 The default value is set to `NONE` to use it as a default `when the inplace option is turned off` or `when the inplace option is turned on but it's a unary operation`, then there's no need to express direction using `LEFT` or `RIGHT`. However, if expressing it this way causes confusion, it would be fine to leave only `LEFT` and `RIGHT`.

If so, what about adding the explanation to the comment?

```suggestion
  NONE,  /**< default value; the inplace option is turned off */
```

Reply:
For a clearer explanation, the default value of the inplace direction is `None`, and it will be set to 'LEFT' or 'RIGHT' only when the type of the operation layer is `binary` and the `is_inplace` setting is `true`.

Reply:
Oh I see. Thank you for the detail.